### PR TITLE
Review request of my ground work.

### PR DESCRIPTION
**Pull request by spidey1**
I've completed as much as I can. I ran across a couple of issues, which I have marked as comments in the code:

views/index.erb : Line 197,
views/index.erb : Line 203
and the commented div in views/index.erb : Line 2, I wanted this div to show when the user pressed the "Update profile" button.

### CODE REVIEW

File: views/index.erb [hunk 1]
```diff
@@ -49,11 +171,42 @@
...
+    }
+
+  });
+
+  // Hardcoded my github url as it would not save in my profile
+  var profileItems = document.getElementsByClassName('item-value');
+
+  for(var i = 0; i < profileItems.length; i++) {
+    if(profileItems[i].innerHTML.toLowerCase() === "spidey1") {
+      profileItems[i].innerHTML="<a href='https://github.com/spidey1'>https://github.com/spidey1</a>"
```
Comment (dmgarland) [inline views/index.erb]: OK I think we're putting in some server side logic to clean off the protocol, maybe that's affecting it?

File: views/index.erb [hunk 2]
```diff
@@ -49,11 +171,42 @@
...
+    var number = document.getElementById('upper-number').value;
+    var resultField = document.getElementById('primes-result');
+    console.log(number);
+
+    if (isNaN(number) || number < 2) {
+      resultField.innerHTML = "Invalid number, please enter a number larger than 1"
+    }
+    else {
+      //Could not figure this bit out, I need to make some sort of server request to call the prime fucntion.
+      resultField.innerHTML = "Could not figure this bit out, I need to make some sort of server request to call the prime fucntion";
```
Comment (dmgarland) [inline views/index.erb]: Yeah you don't need to link this from your profile that's fine. You could put in some examples of it being used with <%= %>

```
<%= Primes.sum_to(100) %>
```

**Comment by dmgarland** [comment]
Good stuff! We'll be in touch

**Comment by spidey1** [comment]
Hi Dan, thanks for the feedback, I'll try out your suggestions.

**Pull request closed without merging**